### PR TITLE
Solution Andriy Bahayev

### PR DESCRIPTION
**Pull request by ABfeMate**
- [DEMO LINK](https://ABfeMate.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ABfeMate.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]
please deploy to GitHub pages (npm run deploy)

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Remove default visited styles for links
and remove title...

**Comment by ABfeMate** [comment]
For some reason I cannot push send my code. I see this error message.
image

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
`git push origin develop -f`
If you have some troubles you can describe problem in fe_chat or your personal chat with mentors. You have no need to wait for review if solution is not ready

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,71 @@
...
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
```
Comment (marina-tilniak) [inline src/index.html:24]: there shouldn’t be empty line between child and parent element

File: src/style.css [hunk 2]
```diff
@@ -1 +1,70 @@
...
+  text-decoration: none;
+}
+
+.nav__link:hover,
+.nav__link:active,
+.nav__link.is-active:link {
+  color: #00acdc;
+}
+
+.nav__list .nav__item:first-child .nav__link {
```
Comment (marina-tilniak) [inline src/style.css:57]: don't use so much specificity

File: src/style.css [hunk 3]
```diff
@@ -1 +1,70 @@
...
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__list .nav__link {
```
Comment (marina-tilniak) [inline src/style.css:32]:
```suggestion
.nav__link {
```

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,70 @@
...
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+            href="#laptops"
+            class="nav__link"
+            data-qa="hover"
+            >
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:44]:
```suggestion
            <a
              href="#laptops"
              class="nav__link"
              data-qa="hover"
            >
```

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Great job 👏 
Keep going!

**Pull request closed without merging**